addons/mail/static/tests/discuss/core/web/command_palette_tests.js

```diff
@@ -28,6 +28,34 @@ QUnit.test("open the chatWindow of a user from the command palette", async (asse
     assert.containsOnce($, ".o-mail-ChatWindow");
 });
 
+QUnit.test("basic rendering: partner displayed in the command palette", async (assert) => {
+    const { advanceTime } = await start({ hasTimeControl: true });
+    triggerHotkey("control+k");
+    await nextTick();
+    // Switch to partners
+    await editSearchBar("@");
+    await afterNextRender(() => advanceTime(commandSetupRegistry.get("@").debounceDelay));
+    assert.containsN($, ".o_command img", 2);
+    assert.containsN($, ".o_command .o-mail-ImStatus", 2);
+    assert.containsOnce($, ".o_command:contains('Mitchell Admin')");
+    assert.containsOnce($, ".o_command:contains('OdooBot')");
```

Comment:
Honestly `img` and `o-mail-ImStatus` is not really important. I would just drop them, because they are not that useful for this test.
The test want to make sure we show the right suggestions when typing `@`. The style of items can be their own tests, or really no test, as it's pretty easy to see and adapt accordingly (and also assertion on specific classnames or tag doesn't really ensure the style is broken)